plugins/modules/mso_schema_validate.py

```diff
@@ -19,6 +19,7 @@
 - This module is used to verify if a schema can be deployed to site without any error.
 - This module can only be used on versions of MSO that are 3.3 or greater.
 - Starting with MSO 3.3, the schema modules in this collection will skip some validation checks to allow part of the schema to be updated more easily.
+- DEPRECATION for NDO 4.0, Validation is executed by NDO 4.0 and higher upon request.
```

Comment:
DEPRECATED for NDO v4.0 and later. Use M(cisco.mso.ndo_schema_template_validate) on NDO v4.1 and later.